lib/Mutators/AndOrReplacementMutator.cpp

```diff
@@ -588,9 +588,9 @@ AND_OR_MutationType AndOrReplacementMutator::findPossibleMutationInBranch(
   return AND_OR_MutationType_None;
 }
 
-std::vector<MutationPoint *>
-AndOrReplacementMutator::getMutations(Bitcode *bitcode,
-                                      llvm::Function *function) {
+std::vector<MutationPoint *> AndOrReplacementMutator::getMutations(
+    Bitcode *bitcode, llvm::Function *function,
+    const mull::InstructionFilter &instructionFilter) {
```

Comment:
All Mutators now get `instructionFilter` as a parameter. It is very important job that `instructionFilter` does, so I would be fine keeping it like this in case there is no better solution to pass it to Mutator in some other way.